synapse/models/infotech.py

```diff
@@ -25,559 +24,770 @@ class SemverType(DataType):
+                    }),
+                    ('host', ('it:host', {}), {
+                        'doc': 'The host running the process that created the mutex. Typically the same host referenced in :proc, if present.',
+                    }),
+                    ('exe', ('file:bytes', {}), {
+                        'doc': 'The specific file containing code that created the mutex. May or may not be the same :exe specified in :proc, if present.',
+                    }),
+                    ('time', ('time', {}), {
+                        'doc': 'The time the mutex was created.',
+                    }),
+                    ('name', ('it:dev:mutex', {}), {
+                        'doc': 'The mutex string.',
+                    }),
                 )),
-
-                ('it:exec:file:del', {'ptype': 'guid'}, (
-                    ('proc', {'ptype': 'it:exec:proc', 'doc': 'The main process executing code that deleted the file.'}),
-                    ('host', {'ptype': 'it:host',
-                         'doc': 'The host running the process that deleted the file. Typically the same host referenced in :proc, if present.'}),
-                    ('exe', {'ptype': 'file:bytes',
-                         'doc': 'The specific file containing code that deleted the file. May or may not be the same :exe specified in :proc if present.'}),
-                    ('time', {'ptype': 'time', 'doc': 'The time the file was deleted.'}),
-                    ('path', {'ptype': 'file:path', 'doc': 'The path where the file was deleted.'}),
-                    ('path:dir', {'ptype': 'file:path', 'doc': 'The parent directory of the file path (parsed from :path).', 'ro': 1}),
-                    ('path:ext', {'ptype': 'str:lwr', 'doc': 'The file extension of the file name (parsed from :path).', 'ro': 1}),
-                    ('path:base', {'ptype': 'file:base', 'doc': 'The final component of the file path (parsed from :path).', 'ro': 1}),
-                    ('file', {'ptype': 'file:bytes', 'doc': 'The file that was deleted.'}),
+                ('it:exec:pipe', {}, (
+                    ('proc', ('it:exec:proc', {}), {
+                        'doc': 'The main process executing code that created the named pipe.',
+                    }),
+                    ('host', ('it:host', {}), {
+                        'doc': 'The host running the process that created the named pipe. Typically the same host referenced in :proc, if present.',
+                    }),
+                    ('exe', ('file:bytes', {}), {
+                        'doc': 'The specific file containing code that created the named pipe. May or may not be the same :exe specified in :proc, if present.',
+                    }),
+                    ('time', ('time', {}), {
+                        'doc': 'The time the mutex was created.',
+                    }),
+                    ('name', ('it:dev:mutex', {}), {
```

Comment:
Should this be an `it:dev:pipe`? it was before

Reply:
good catch